docs/reference/mount.md

```diff
@@ -127,15 +157,39 @@ ls -l "./pets/dogs/"
 find ./pets -name "*.small.jpg"
 open -a Preview "./pets/dogs/golden_retrievers/cute.jpg"
 everest umount "./pets"
-
 ```
+
 #### Working with Data Locally
 Mount the remote lakeFS server and use all familiar tools without changing the workflow.
 ```bash
-everest mount "lakefs://image-repo/main/datasets/pets/" "./pets"
-pytorch_train.py --input "./pets"
+everest mount lakefs://image-repo/main/datasets/pets/ ./pets
+pytorch_train.py --input ./pets
 duckdb "SELECT * FROM read_parquet('pets/labels.parquet')"
-everest umount "./pets"
+everest umount ./pets
+```
+
+### Write Mode
```

Comment:
I think this should be it's own section i.e `##` not nested under examples 
Please add some context here and link to semantics section

Reply:
As I see it, it is supposed to look like this:
Examples:
  Read-mode:
    Example1:
    Example2:
  Write-mode:
    Example1:
    ...